server/source/lua/LuaMod.cpp

```diff
@@ -142,14 +147,82 @@ void LuaMod::registerSmeltingRecipe(const sol::table &table) {
 	Registry::getInstance().registerRecipe<SmeltingRecipe>(input, output);
 }
 
+void LuaMod::registerTree(const sol::table &table) {
+	std::string stringID = m_id + ":" + table["id"].get<std::string>();
+	std::string label = table["name"].get<std::string>();
+
+	Tree &tree = Registry::getInstance().registerTree<Tree>(stringID, label);
+	tree.setLogBlockID(Registry::getInstance().getBlockFromStringID(table["log_block"]).id());
+	tree.setLeavesBlockID(Registry::getInstance().getBlockFromStringID(table["leaves_block"]).id());
+}
+
+void LuaMod::registerBiome(const sol::table &table) {
+	std::string stringID = m_id + ":" + table["id"].get<std::string>();
```

Comment:
You're adding the mod ID even though you explicitly stated it in Lua.

In the Lua file for biomes, instead of `default:mountain` ID you should just have `mountain`